pkg/api/decode_aws.go

```diff
@@ -17,6 +17,9 @@
 
 package api
 
+import "github.com/mariomac/pipes/pkg/graph/stage"
+
 type DecodeAws struct {
+	stage.Instance
```

Comment:
https://github.com/mariomac/pipes/tree/main/examples/tutorial/02-highlevel-nodes#defining-providers-for-each-node